spec/features/users/user_can_create_folder_spec.rb

```diff
@@ -22,5 +22,28 @@
 
       expect(current_path).to eq(Folder.last.url)
     end
+    it "User can create a public folder" do
```

Comment:
is there a sad path test to add here? what happens if the folder isn't created correctly?